### PR TITLE
Fix secondaryFile as single string/expression in CWL1.0

### PR DESCRIPTION
**Pull request by bogdang989**
Fixes https://github.com/common-workflow-language/cwltool/issues/1110

### CODE REVIEW

**Comment by mr-c** [comment]
Jenkins, test this please

File: cwltool/builder.py [hunk 1]
```diff
@@ -313,6 +313,10 @@ def _capture_files(f):
                         else:
                             sf_required = True
 
+                        # Convert to secondaryFileSchema for cw1.0 tools
+                        if isinstance(sf, str):
+                            sf = {"pattern": sf}
```
Comment (mr-c) [inline cwltool/builder.py]: I think this indicates an error in the v1.1 schema

File: tests/test_examples.py [hunk 2]
```diff
@@ -805,6 +805,21 @@ def test_secondary_files(self):
         assert "completed success" in stderr
         assert error_code == 0
 
+    def test_secondary_files_v1_0(self):
+        test_file = "secondary-files-string-v1.cwl"
+        test_job_file = "secondary-files-job.yml"
+        try:
+            old_umask = os.umask(stat.S_IWOTH)  # test run with umask 002
+            error_code, _, stderr = get_main_output(
+                ["--enable-dev",
```
Comment (mr-c) [inline tests/test_examples.py]: why `--enable-dev` here?

**Comment by bogdang989** [comment]
Hi @mr-c  thanks for reviewing. I took a closer look and found that the proper place to fix this would be in update.py

File: tests/test_examples.py (hunk 2 above)
Comment (bogdang989) [reply tests/test_examples.py]: Yup, that's unnecessary :)

**Comment by tetron** [comment]
Thanks!

**Pull request merged**